### PR TITLE
add js-shims

### PR DESCRIPTION
**Pull request by caub**
It's a bit annoying because js-shims seem to alter the harmony behavior

when loading `airbnb-js-shiims/target/es2019` then this test fails:
```
● jsEvalPlugin › exposes unstable harmony features with h>
    Expected value to equal:
      "(okay) 3n"
    Received:
      "Error"
```

I'm trying to isolate the package which cause the bug

### CODE REVIEW

**Comment by ljharb** [comment]
Wait, what? How are they breaking anything? What is h>?

**Comment by ljharb** [comment]
And harmony is broken by design - why do those flags matter?

**Comment by caub** [comment]
h> (harmony) is using harmony and experimental flags https://github.com/brigand/jellobot/blob/master/src/plugins/js-eval/Dockerfile#L52-L60

if we pass the [`stable=true`](https://github.com/brigand/jellobot/blob/master/src/plugins/js-eval/jsEval.js#L8) option, that default command is overriden

But yes, after thoughts you're right, babel does all those things already, it's not worth the trouble to have h>

I'll remove most flags (might just keep --experimental-vm-modules and --experimental-modules for m>, they don't hurt

Thanks

**Comment by ljharb** [comment]
Please do file upstream issues tho, each of my shims should still work correctly with harmony flags enabled.

Also, please do include airbnb-js-shims in the bot when it makes sense :-)

**Comment by caub** [comment]
@ljharb updated, now the tests pass

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jellobot",
-  "version": "0.0.0",
+  "version": "1.0.0",
```
Comment (ljharb) [inline package.json]: typically PRs don’t update the version of things

Comment (caub) [reply package.json]: fixed

**Comment by brigand** [comment]
There's `nvm ls-remote` which gives all node.js versions without using a token. We could either call that, or extract the relevant code/algorithm for use here.

**Comment by ljharb** [comment]
Yeah I’m not sure why we’d need to talk to github when nvm can do everything on the cli.

**Comment by caub** [comment]
I don't have nvm installed for example, so for now I've just removed that part, and have an environment variable `NODE_VERSION` that should be defined before building the image with `src/plugins/js-eval/init` script, so  `NODE_VERSION=$(nvm ls-remote latest) src/plugins/js-eval/init` or something with nvm

ps: one way to fetch the version without tokens: `curl -s https://nodejs.org/dist/ | awk '/href="v11./{ split($2, a, "[<>/]"); print a[3] }' | sort -V | tail -n1`

**Comment by ljharb** [comment]
Why not install nvm in the container?

**Comment by caub** [comment]
nice idea, I didn"t manage to make it work yet
```
FROM debian:9-slim
RUN apt-get update && apt-get install -y ca-certificates curl wget gnupg dirmngr xz-utils --no-install-recommends
RUN curl https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
RUN export NVM_DIR="$HOME/.nvm" && \. "$NVM_DIR/nvm.sh"
RUN nvm --version
# ^ gives /bin/sh: 1: nvm: not found
```
if the nvm way doesn't make it, I might try to improve on the https://nodejs.org/dist/ parsing way

**Comment by caub** [comment]
Yes you're right, I'm not far to have a working image using nvm

```
FROM debian:9-slim
RUN apt-get update && apt-get install -y ca-certificates curl --no-install-recommends \
 && curl -s https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash \
 && \. /root/.nvm/nvm.sh && nvm install node

ENTRYPOINT ["bash", "-c", "\. /root/.nvm/nvm.sh"]

CMD ["node"]
```

not there, yet

in any case, we can make it a separate PR if you don't mind

**Pull request merged**